recipes/lpub3d/recipe.yaml

```diff
@@ -0,0 +1,113 @@
+schema_version: 1
+
+context:
+  lp3d_name: lpub3d
+  lp3d_version: 2.4.9.86.4133
+  lp3d_sha: 8880540b
+  lp3d_ldview: ldview
+  lp3d_povray: povray
+  lp3d_ldglite: ldglite
+  lp3d_3rdparty_dir: ${{ 'lpub3d_windows_3rdparty' if win }}
+
+package:
+  name: ${{ lp3d_name|lower }}
+  version: ${{ lp3d_version }}
+
+source:
+  - url: https://github.com/trevorsandy/${{ lp3d_name }}/archive/${{ lp3d_sha }}.tar.gz
+    sha256: 875a1da26bda9f8cf191fb4064897aa9196db4f55103eb2daf4d21a843bae729
+    target_directory: ${{ lp3d_name }}
+
+  - url: https://github.com/trevorsandy/${{ lp3d_ldview }}/archive/lpub3d-build.tar.gz
+    sha256: f8d5db81e65d4372b302610f686b50f3ed17797ac724cd968d9a10a942e11db0
+    target_directory: ${{ lp3d_ldview }}
```

Comment:
Conda claims that the hash of this archives has changed. This is caused by one/both of two problems. GitHub archives hashes are not stable over a period of months OR this url points to a tag that has moved. The first issue is a known issue that Github will not fix. The work around is that you have to attach your own archive to a release because user uploaded files are static unlike GitHub generated archives which are created on demand and then cached for only a few months.